src/answered_question/urls.py

```diff
@@ -15,4 +15,7 @@
 
 urlpatterns = [
     path('', include(router.urls)),
+    path('answeredquestions/bulk',
+         AnsweredQuestionViewSet.as_view({'post': 'bulk_create'}),
+         name='create-answeredquestions'),
```

Comment:
Update for readability purposes
```suggestion
    path('answered-questions/bulk',
         AnsweredQuestionViewSet.as_view({'post': 'bulk_create'}),
         name='create-answered-questions'),
```